.devcontainer/Dockerfile

```diff
@@ -0,0 +1,11 @@
+# OS
+ARG VARIANT=bullseye
+FROM --platform=linux/amd64 mcr.microsoft.com/vscode/devcontainers/base:0-${VARIANT}
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive && apt-get install -y firefox-esr
+RUN sudo apt-get update
+RUN sudo apt-get install -y libgtk-3-dev
+RUN sudo apt-get install -y python3 python3-pip
+
+RUN pip install poetry
```

Comment:
I notice the CI/GitHub Actions builds run against multiple versions of python - I find it is useful to have all those version installed in the dev env using pyenv so you can test each locally. But this does slow down the docker build a fair bit as it will install each version of python and build from scratch. Something like this:

```dockerfile
# use pyenv to install all the other versions as the vscode user
ARG PYTHON_VERSIONS="3.9 3.10 3.11"
USER vscode
ENV PYENV_ROOT="/home/vscode/.pyenv"
ENV PATH="$PYENV_ROOT/bin:$PATH"
RUN curl https://pyenv.run | bash
RUN $PYENV_ROOT/bin/pyenv install -v ${PYTHON_VERSIONS}
```

Reply:
> run against multiple versions of python - I find it is useful to have all those version installed in the dev env using pyenv so you can test each locally. But this does slow down the docker build a fair bit as it will install each version of python and build from scratch.

oof, I just ran this locally and it was painfully slow. I'd actually suggest we only stick to one version in the dev container and let CI test the others for us for now? We don't have tox or anything like that set up to test all versions locally yet anyways, so you'd have to manually switch and poetry install / test every time. Other thoughts on this?

Reply:
Yeah it was surprisingly slow, like 30+ minutes.  I think we should get rid of it as well.